types/statsd-client/index.d.ts

```diff
@@ -79,24 +88,53 @@ interface HttpOptions extends CommonOptions {
     method?: string;
 }
 
+interface ExpressMiddlewareOptions {
+    /**
+     * Metric name to use for reporting if a matching route is not
+     * found (default "unknown_express_route").
+     */
+    notFoundRouteName?: string;
+
+    /**
+     * Optional callback called after reporting metrics for an
+     * express route.
+     */
+    onResponseEnd?: (client: StatsdClient, startTime: Date, req: express.Request, res: express.Response) => void;
+
+    /**
+     * Enables inclusion of per-URL response code and timing
+     * metrics (default false).
+     */
+    timeByUrl?: boolean;
+}
+
 declare class StatsdClient {
     constructor(options: TcpOptions | UdpOptions | HttpOptions);
 
-    counter(metric: string, delta: number): void;
-    increment(metric: string, delta?: number): void;
-    decrement(metric: string, delta?: number): void;
+    counter(metric: string, delta: number, tags?: {}): this;
```

Comment:
Any type is assignable to `{}`, including primitives like `number`. So this is the same as saying `tags?: any`. To be clearer, I suggest changing the type to `any` (or `object`, if your intention is to disallow primitive types).

Reply:
Updated to a real hash type.